src/summary/cards/horizontal-section-card.css

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
same applies for this, see comment above. is it needed?

Reply:
You are right. We don't need